### ISSUE TITLE
If a build workflow fails, activated license is retained by BitRise

### ISSUE THREAD

**Issue by lakario** [bitrise-steplib/steps-xamarin-user-management#4]
As a result of a BitRise build failing, the xamarin-user-management step claimed a license seat, but never released it. 

```
You have reached the maximum number of activations.
ERRO[22:15:00] Step (xamarin-user-management@0.9.1) failed, error: (exit status 1)
```

I was able to resolve this by manually deactivating BitRise in our Xamarin account.

The default workflow configuration for Xamarin should always release the license, regardless of failure. Alternatively, every attempt to login to Xamarin should be pre-empted by a logout.

**Comment by birmacher** [comment]
Hi @lakario!

Most of the time Bitrise will free it up, but there are a couple of cases when the workflow execution halts. ( Aborted build )

To solve this issue we will move the license de-registration to the website logic from the workflow.
I've added this to our roadmap and we will release it in the next weeks.

**Comment by birmacher** [comment]
Fix is coming really shortly: #6 with Xamarin.Mac support ✨

**Comment by birmacher** [comment]
The new `1.0.0` version is available on Bitrise. Please update your step to the new version and it should resolve the issue.
You can also remove the `logout` step at the end of your workflow. We will do it in the background for you.

I'm closing this issue, but if you find any problems just reopen it.
